src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -233,7 +234,7 @@ def delete_key(cmd,
     if not_deleted_entries:
         if deleted_entries:
             logger.error('Deletion operation partially succeed. Some keys are not successfully deleted. \n %s',
```

Comment:
do you know if %s formatting will support non-ascii characters?

Reply:
It does. It just the json.dump() has a default argument that converts all characters to ascii that we need change.